### ISSUE TITLE
Selecting Wifi submenu makes system menu disappear

### ISSUE THREAD

**Issue by DoughMucker** [Rayzeq/quick-settings-audio-panel#46]
**Describe the bug**
Clicking the wifi sub-menu right arrow to see a list of wifi networks results in the system menu disappearing. Clicking on where the system menu should be results in the menu reappearing (you can see the animation of the list of wifi networks being collapsed). If Dash to Panel is disabled and then re-enabled (with SQAP on) you won't see the bug. If SQAP is disabled and then re-enabled (with Dash to Panel on) then you'll see the bug.

**Compatibility**
Does this issue arise when all other extensions are disabled ?
It seems to be an incompatibility with Dash to Panel, as disabling that extension (or SQAP) results in the correct behaviour. 

**Screenshots**
Press the Super key and that is what the screen would look like when the wifi sub-menu is selected.

**Desktop (please complete the following information):**
Fedora 40
 - Gnome shell version: 46

**Additional context**

**Comment by Rayzeq** [comment]
I've tested things with Dash to Panel, and the quick settings are completely broken when the panel is aligned to the left while the taskbar is to the right of the screen (and vice versa), however, the wifi submenu never disappeared, could you send screenshots before and after the problem (or better, a screen recording) ?

**Comment by DoughMucker** [comment]
https://github.com/Rayzeq/quick-settings-audio-panel/assets/14085399/fa0efa2a-5cce-483b-bf71-f36313930067

Here's a screencast of it. The recording doesn't show my mouse though. Just a few clicks...I enable QSAP (if Dash to Panel is enabled after QSAP, then you won't see it), click the dash/panel to bring up the system status area, click the right arrow next to wifi (system status area disappears), I wait a bit and click in the same spot where the arrow was (system status area reappears).

**Comment by DoughMucker** [comment]
Odd...I saw that I still had GSConnect enabled. So I disabled it and the issue went away.  Enabled GSConnect and the problem came back. I thought I had disabled all extensions when trying to isolate the issue, but perhaps not. So the issue seems to be with GSConnect.

**Comment by Rayzeq** [comment]
I've tried to reproduce your setup, but I'm still unable to reproduce the issue (see attached video), and I have absolutely no idea what's causing it. I'm not even sure if GSConnect is related to this issue, I assume you've disabled it and directly tried to reproduce the issue, in which case it's possible that it was changing something (removing a button) in the quick settings panel that solved the issue.

The only idea I have left is to try and reproduce the issue with your bar at the top of the screen to see if it behaves differently, which might tell me the cause of the issue.

[Screencast from 2024-06-19 13-15-27.webm](https://github.com/Rayzeq/quick-settings-audio-panel/assets/26172375/c721517e-0d2f-49c8-9b44-22aef44feb2e)

**Comment by DoughMucker** [comment]
I put the bar at the top of the screen and could not reproduce the issue. Put it back and the bottom and the issue was back. It's all good though, because I don't use GSConnect that much and can disable the extension when not in use so I won't experience the issue. 

Thank you for you help and for your time making the extension!

**Comment by DoughMucker** [comment]
Weirdness...
I'm thinking the issue is related to Dash to Panel (or perhaps, more fundamentally, Gnome). The main difference I saw between your video and mine was the height/number of the quick settings tiles...I installed the Compact Quick Settings extension and disabled all other extensions and still saw the issue. The Wi-Fi setting was 7th from the bottom. The issue seems to occur when Wi-Fi is the 6th (or greater) item from the bottom. Oddly, when I use Super-S to bring up the quick settings, I do not experience the issue, whether Wi-Fi is 6 or 7 tiles from the bottom but using the mouse to bring up the panel I'll see the issue. So I'll close this issue.
